src/charactersheet/viewmodels/character/magic_items/index.html

```diff
@@ -62,7 +64,7 @@
       <!-- /ko -->
       <!-- ko if: filteredAndSortedMagicItems().length == 0 -->
         <tr class="clickable">
-          <td data-toggle="modal" data-target="#addMagicItem" id="magicItemsAddIcon" href="#"
+          <td data-toggle="modal" data-target="#addMagicItem" href="#"
```

Comment:
Why was the ID removed from here? This add button should also be tested.

Reply:
That was intentional, there are 2 places to add. This id removed was intended for the other location.